### DISCUSSION TITLE
SSL defined, can't connect from Android device

### DISCUSSION THREAD

**Discussion by Concubine** [jellyfin/jellyfin#8987] [Q&A]
Hello and thank you for a wonderful product!

I'm here as I have setup a container with podman, it is currently configured to run and linger and I don't seem to have any issue with using it none secure.

Once I enable a certifiate, have generated with the [guide here](https://jellyfin.org/docs/general/networking/index.html), I can't seem to be able to use via the mobile application and was wondering if it is related to the fact my certificate is self signed? if so - is there an option to tell the app to not validate the certifiate so that I can use the self signed one or is the only solution to get a propely signed certificate?

If the latter is the answer, can I use certbot for this purpose?

EDIT:

Tried importing my certificate to the phone, still won't allow me to reach the https address via the application.

**Comment by sjorge** [comment]
You might also be bitten by https://github.com/jellyfin/jellyfin/issues/8340
